democracy/src/lib.rs

```diff
@@ -114,22 +156,11 @@ pub mod pallet {
 	pub(super) type Tallies<T: Config> =
 		StorageMap<_, Blake2_128Concat, ProposalIdType, Tally, OptionQuery>;
 
-	#[pallet::storage]
-	#[pallet::getter(fn vote_entries)]
-	pub(super) type VoteEntries<T: Config> = StorageDoubleMap<
-		_,
-		Blake2_128Concat,
-		ProposalIdType,
-		Blake2_128Concat,
-		VoteEntry<T::AccountId>,
-		(),
-		ValueQuery,
-	>;
 	// TODO set default value
```

Comment:
is this still a TODO? and does the follwoing need to be ValueQuery? Whenever possible I prefer OptionQuery

Reply:
ok option query now